smoke-tests/smoke-sdk-http.bats

```diff
@@ -3,36 +3,46 @@
 load test_helpers/utilities
 
 CONTAINER_NAME="app-sdk-http"
+COLLECTOR_NAME="collector"
 OTEL_SERVICE_NAME="otel-python-example"
+TRACER_NAME="hello_world_tracer"
+METER_NAME="hello_world_meter"
 
 setup_file() {
 	echo "# 🚧" >&3
-	docker-compose up --build --detach collector ${CONTAINER_NAME}
-	wait_for_ready_app ${CONTAINER_NAME}
-	curl --silent "http://localhost:5000"
+	docker-compose up --build --detach collector
+	wait_for_ready_collector ${COLLECTOR_NAME}
+	docker-compose up --build --detach ${CONTAINER_NAME}
 	wait_for_traces
+	wait_for_metrics
 }
 
 teardown_file() {
-	cp collector/data.json collector/data-results/data-${CONTAINER_NAME}.json
+  cp collector/data.json collector/data-results/data-${CONTAINER_NAME}.json
 	docker-compose stop ${CONTAINER_NAME}
 	docker-compose restart collector
 	wait_for_flush
 }
 
 # TESTS
 
-@test "Manual instrumentation produces span with name of span" {
-	result=$(span_names_for ${OTEL_SERVICE_NAME})
-	assert_equal "$result" '"sleep"'
+@test "Manual instrumentation produces parent and child spans with names of spans" {
+	result=$(span_names_for ${TRACER_NAME})
+	assert_equal "$result" '"world"
+"hello"'
 }
 
 @test "Manual instrumentation adds custom attribute" {
-	result=$(span_attributes_for ${OTEL_SERVICE_NAME} | jq "select(.key == \"delay_ms\").value.intValue")
-	assert_equal "$result" '"100"'
+	result=$(span_attributes_for "${TRACER_NAME}" | jq "select(.key == \"message\").value.stringValue")
+	assert_equal "$result" '"hello world!"'
+}
+
+@test "BaggageSpanProcessor: key-values added to baggage appear on child spans" {
+	result=$(span_attributes_for ${TRACER_NAME} | jq "select(.key == \"for_the_children\").value.stringValue")
```

Comment:
just a note: is there somewhere in devleoping.md we should add a Smoke Tests section with the Make Commands and dependencies needed (brew install bats-core, brew install jq)  

When I was going to run this for the first time it failed because I had dependencies to install.

Reply:
👍  I added these notes into the `DEVELOPING.md`